datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -4040,6 +4101,31 @@ input TagUpdateInput {
     ownership: OwnershipUpdate
 }
 
+"""
+Arguments provided to update a DataPlatformInstance Entity
+"""
+input DataPlatformInstanceUpdateInput {
+    """
+    Update to ownership
+    """
+    ownership: OwnershipUpdate
+
+    """
+    Update to deprecation status
+    """
+    deprecation: DatasetDeprecationUpdate
+
+    """
+    Update to institutional memory, ie documentation
+    """
+    institutionalMemory: InstitutionalMemoryUpdate
+
+    """
+    Update to tags
+    """
+    tags: GlobalTagsUpdate
+}
+
```

Comment:
Same with holding on this until we build out the update endpoint and add `update` to the `DataPlatformInstanceType`

also for what it's worth, we won't actually be calling a new update endpoint from the UI for any of these fields as they all have their own specific endpoint (ie. `addOwner`, `removeOwner`, `updateDeprecation`, `addLink`, etc.)

But having this exposed via graphql API as an option still makes sense

Reply:
I'm ok with that, addressed in f646ed36d